modules/configuration/src/main/java/org/apache/ignite/configuration/Configurator.java

```diff
@@ -214,9 +205,13 @@ public T getRoot() {
         property.addListener(new PropertyListener<PROP, PROP>() {
             /** {@inheritDoc} */
             @Override public void update(PROP newValue, ConfigurationProperty<PROP, PROP> modifier) {
-                storage.save(key, newValue);
+//                storage.save(key, newValue);
```

Comment:
Yeah, about these listeners... I guess they'll be completely removed from the code, we'll discuss this later as well